### ISSUE TITLE
Package.getStylesheetsPath is deprecated.

### ISSUE THREAD

**Issue by sojan-official** [nesQuick/atom-wordcount#33]
Store package style sheets in the `styles/` directory instead of `stylesheets/` in the `wordcount` package
```
Package.getStylesheetsPath (/Applications/Atom.app/Contents/Resources/app/src/package.js:433:9)
Package.getStylesheetPaths (/Applications/Atom.app/Contents/Resources/app/src/package.js:444:32)
```

**Comment by nesQuick** [comment]
I'm happy to take your pull requests to get this fixed :)

**Comment by nesQuick** [comment]
fixed by #34 

thanks a lot for your help! :heart:

**Comment by sojan-official** [comment]
glad to chip in :smile:
